website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,46 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+Multi-selection enables users to select multiple rows in a table, facilitating bulk operations, actions, and editing. 
+
+Multi-selection consist of: 
+
+1. `Header Column / Select`: Used in the header row of the table. This acts as a master checkbox, enabling end-users to either select or deselect all rows on a single page simultaneously.
+
+![Example of multi-selection in a table header](/assets/components/table/multi-select-header.png)
+
+2. `Cell / Select`: Used in each table row. It allows for individual row selection and aligns with the Header Column component, reflecting the overall selection status of the table.
+
+![Example of multi-selection within table cells](/assets/components/table/multi-select-cells.png)
+
+!!! Info 
+
+In the Ember component, unlike the `cell / select` Figma component, achieving an indeterminate state is not possible. 
```

Comment:
Repeating my comment from the old PR:

_@andgen404 I don't quite understand what you mean here. Are you talking about having an indeterminate state on the checkboxes in the table body rows (vs. in the table header)? Because that doesn't make sense to do anyway. I would be very confused by this as a consumer._

I would delete this entire Info alert.

Reply:
Yeah, that's what I'm trying to convey, but I agree. I think calling it out here doesn't make much sense.